akka-actor-typed-tests/src/test/java/jdocs/akka/typed/InteractionPatternsTest.java

```diff
@@ -49,7 +48,46 @@ public PrintMe(String message) {
       .onMessage(EnableOutput.class, (ctx, enableOutput) -> enabledPrinterBehavior())
       .build();
   }
-  // #fire-and-forget
+  // #fire-and-forget-definition
+
+  // #request-response-protocol
+  interface Protocol {}
```

Comment:
For me the term protocol means the full set of messages, and semantics. Here it's use only for the incoming messages (requests). I think `Request` and `Response` would be enough here.

Reply:
Protocol; 
Request extends Protocol; 
Response extends Protocol;
SpecificResponse extends Response;
...


Is how I think about it (i.e. I agree), and in such a simple one you can just say response/request I think, without complicating it too much